Packs/CommonScripts/ReleaseNotes/1_15_57.md

```diff
@@ -0,0 +1,7 @@
+
+#### Scripts
+
+##### New: AnalyzeTimestampIntervals
+
+- New: This script is designed to analyze a series of timestamps in Unix (Epoch) format, by calculating the intervals between them and determining the consistency of those intervals. It starts by computing the differences between each consecutive timestamp and then calculates the differences between these intervals. The script then evaluates the statistical properties of the intervals, such as the mean, median, and standard deviation. Based on these calculations and a user-defined or default threshold, it determines whether the pattern of intervals is likely to be automated and if it is consistent.<~XSIAM> (Available from Cortex XSIAM %%XSIAM_VERSION%%).</~XSIAM>
```

Comment:
1. replace %%XSIAM_VERSION%% with 8.7.0 or remove this (tbh not sure if this is required or not)
2. I think it's TMI, especially for release notes - stick to the use-case, what is it used for?